cylc/flow/scripts/workflow_state.py

```diff
@@ -370,7 +369,7 @@ def main(parser: COP, options: 'Values', *ids: str) -> None:
         [
             options.depr_task,
             options.depr_status,
-            options.depr_msg,  # --message and --trigger
```

Comment:
Comment was wrong and isn't really necessary anyway.
There was never any `--trigger` option; it is `--output` and `--message` (singular) that are deprecated.